xapian-applications/omega/docs/omegascript.rst

```diff
@@ -337,6 +337,19 @@ $map{LIST,STUFF)
 	versions). If the tabs are a problem, use $list{$map{...},}
 	to get rid of them.
 
+$match{REGEX,STRING[,OPTIONS]}
+	peform a regex match using Perl-compatible regular expressions. This
+	command will return true if a match is found
```

Comment:
You need to say what happens if this doesn't match, but also I'd get rid of 'this command will…'. Something like:

> Return true if matches using Perl-compatible regular expressions. Else return an empty string.